h/views/api_auth.py

```diff
@@ -20,6 +20,7 @@ def access_token(request):
         'access_token': token.value,
         'token_type': 'bearer',
         'expires_in': TOKEN_TTL.total_seconds(),
+        'refresh_token': token.refresh_token,
```

Comment:
I'm thinking we should maybe only set the `refresh_token` key when the value of it is not `None` or empty. A good example is that the way the test is written, the response will include: `"refresh_token": null`.

Maybe something along the lines of:
```python
payload = {
    'access_token': token.value,
    'token_type': 'bearer',
    'expires_in': TOKEN_TTL.total_seconds()
}
if token.refresh_token is not None:
    payload['refresh_token'] = token.refresh_token
return payload
```